### ISSUE TITLE
Surface heat flux processes need documentation

### ISSUE THREAD

**Issue by brian-rose** [brian-rose/climlab#59]
The climlab processes `climlab.surface.SensibleHeatFlux()` and `climlab.surface.LatentHeatFlux()` are out-of-date and inconsistent with the climlab API. 

These classes need to properly define inputs and diagnostics. It should be possible to set input values *as input arguments* at creation time, like other processes.

Also, the [documentation for these classes](http://climlab.readthedocs.io/en/latest/api/climlab.surface.turbulent.html) is completely useless.

**Comment by brian-rose** [comment]
Finally made some progress on this.

As of e022efa there is now [some useful documentation of surface fluxes](https://climlab.readthedocs.io/en/land/api/climlab.surface.turbulent.html)

Also the classes now have proper diagnostics and inputs declared. Once this is merged into master I will close this issue.

**Comment by brian-rose** [comment]
Hopefully the news docs improve the confusion expressed in #103
